build_scripts/webpack-rules.js

```diff
@@ -98,7 +98,7 @@ module.exports = [
     use: ['style-loader', 'css-loader']
   },
   {
-    test: /\.svg$/,
+    test: /\.(svg|png)/,
```

Comment:
This break the neo4j logo on `:play start`. And possibly more places.

Reply:
This is still an issue.
<img width="489" alt="oskarhane-mbpt 2019-09-12 at 08 49 45" src="https://user-images.githubusercontent.com/570998/64760558-50af5c80-d53a-11e9-9fc1-5832ea87af1e.png">